packages/atomic/src/components/facets/atomic-date-range/atomic-date-range.tsx

```diff
@@ -25,4 +26,12 @@ export class AtomicDateRange {
    * Specifies whether or not the end date should be included in the range.
    */
   @Prop() public endInclusive = false;
+
+  public componentWillLoad() {
+    if (dayjs(this.start).isAfter(dayjs(this.end))) {
+      console.error(
+        `Start of date range is after end for ${this.start} to ${this.end}`
+      );
+    }
+  }
```

Comment:
I think this would be a useful check to include in the headless `buildDateRange` utility. Could we place it there and cover the case with a unit test? I also think we should throw an Error with a clear message when this happens.

In the case of Atomic, I believe we've been showing errors in the UI, not logging them in the console? If this is the case elsewhere in the project, could we do that here?

Reply:
Yes errors are usually shown using the atomic-component-error component

Reply:
We should not be doing that check in Atomic at all, hea,

That's purely a "data" problem, which is a Headless concern.